YAML/YAML.sublime-syntax

```diff
@@ -94,20 +94,16 @@ variables:
   _flow_scalar_end_plain_in: |- # kind of the negation of nb-ns-plain-in-line(c) c=plain-in
     (?x:
       (?=
-          \s* $
-        | \s+ \#
-        | \s* : (\s|$)
-        | \s* : {{c_flow_indicator}}
-        | \s* {{c_flow_indicator}}
+          \s+ \#
+        | \s* (?: $ | {{c_flow_indicator}} | : (?:\s|$|{{c_flow_indicator}}) )
```

Comment:
Was the effect on performance here cause by not grouping or by using a catpuring group? Because I find the new grouping much harder to follow (though that could probably be solved by just splitting it over multiple lines).

Reply:
The duplicate `\s*` had most impact.